# This message describes alarms (eg BSD) from a RADAR sensor.
bool LCA_alarm    # Lane Change Assist alarm
bool CVW_alarm    # Collision (Vehicle?) Warning alarm
bool BSD_alarm    # Blind Spot Detection alarm
